fiona/model.py

```diff
@@ -5,10 +5,14 @@
 from enum import Enum
 import itertools
 from json import JSONEncoder
+import reprlib
 from warnings import warn
 
 from fiona.errors import FionaDeprecationWarning
 
+_coord_repr = reprlib.Repr()
+_coord_repr.maxlist = 1
```

Comment:
For real-world features, the coordinates are likely to make up most of the representation. Showing 20+ can easily overflow displays. One is enough to see where the feature is in space, roughly. 6 would be reprlib's default, but that's still a lot of characters if we're dealing with small scale, projected features.